packages/elements/src/components/ino-nav-drawer/ino-nav-drawer.tsx

```diff
@@ -79,17 +90,28 @@ export class NavDrawer implements ComponentInterface {
     if (this.drawerInstance) {
       this.drawerInstance.open = this.open || false;
     }
-
     this.drawerEl.addEventListener('MDCDrawer:closed', this.closeDrawer);
     this.initTabindex('content');
     this.initTabindex('footer');
+
+    this.modifyMobileItems();
   }
 
   disconnectedCallback() {
     this.drawerEl.removeEventListener('MDCDrawer:closed', this.closeDrawer);
     this.drawerInstance?.destroy();
   }
 
+  private modifyMobileItems() {
+    if (!this.isMobile) return;
+
+    const navItems = this.el.querySelectorAll('ino-nav-item');
+
+    navItems.forEach((item) => {
+      item.classList.add('mobile-nav-item');
+    });
+  }
```

Comment:
When transitioning from the mobile view to the desktop view, we also need to remove these classes again. That's currently not done, right?

It's probably a good idea to separate all the logic thats mobile specific into two methods like `activateMobileMode`and `deactivateMobileMode` or something like that.